google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -99,6 +118,16 @@ public String getModuleName() {
     return moduleName;
   }
 
+  @Attribute("appYamlPath")
+  public String getAppYamlPath() {
```

Comment:
Discussed offline; agreed to leave these out of the config and derive them from the facet config, which can be determined by the `moduleName`. If, for some reason, the user has issues with their facet config (e.g. missing `app.yaml`), they will be notified on deployment.